packages/components/pie-assistive-text/src/index.ts

```diff
@@ -13,8 +20,36 @@ const componentSelector = 'pie-assistive-text';
  * @tagname pie-assistive-text
  */
```

Comment:
I think you could add a `* @slot - Default slot ` comment above the class to assist with the `custom-elements.json` file that is generated at build